src/test/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public abstract void doWork();
+    public abstract void stopWork();
+
+
+}
```

Comment:
Please remove the redundant empty lines at the end of the class declaration. According to the checklist, we don't need them after class declaration or method signature.